src/components/App/App.component.jsx

```diff
@@ -1,57 +1,51 @@
-import React, { useLayoutEffect } from 'react';
-import { BrowserRouter, Switch, Route } from 'react-router-dom';
-
-import AuthProvider from '../../providers/Auth';
-import HomePage from '../../pages/Home';
-import LoginPage from '../../pages/Login';
-import NotFound from '../../pages/NotFound';
-import SecretPage from '../../pages/Secret';
-import Private from '../Private';
-import Fortune from '../Fortune';
-import Layout from '../Layout';
-import { random } from '../../utils/fns';
+import React, { useState, useEffect } from 'react';
 
-function App() {
-  useLayoutEffect(() => {
-    const { body } = document;
+import AuthProvider from '../../providers/Auth/Auth.provider';
+import VideoProvider from '../../providers/Video/Video.provider';
+import UserPreferencesProvider from '../../providers/Preferences/UserPreferences.provider';
+import AppTheme from '../AppTheme/AppTheme';
+
+async function initGoogle() {
+  await new Promise((res, rej) => {
+    window.gapi.load('client', { callback: res, onerror: rej });
+  });
 
-    function rotateBackground() {
-      const xPercent = random(100);
-      const yPercent = random(100);
-      body.style.setProperty('--bg-position', `${xPercent}% ${yPercent}%`);
-    }
+  window.gapi.client.setApiKey('AIzaSyA5IW0wuDEdax20lHoe0bi2csRdsybvPTw');
```

Comment:
its better to not have secrets in public repositories, like the password or, in this case, the API key, use `dotenv` node library to extract those values into environment variables
Reference: https://www.npmjs.com/package/dotenv

Reply:
Thank you, I'll check this up! and change it ASAP!

Reply:
Api key was setted on ENV and its no longer exposed on github.